PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,6 @@
+
+*This is a reminder of what a message in a pull request should minimally require. Please, read the guidelines below and **delete** them once you followed the guidelines so that your message only contains the intended message.*
```

Comment:
nitpick: read the guidelines below and ->read the guidelines below**,** and    (two sentences).

nitpick: once you followed the guidelines so that your message only contains the intended message -> so your pull request description only contains the intended message    (concise).

Reply:
Done in 88ef263.